gateway/src/shard/impl.rs

```diff
@@ -244,13 +249,9 @@ impl Shard {
         Ok(())
     }
 
-    /// Shuts down the shard.
+    /// Shuts down the shard
```

Comment:
I don't think this is needed, we don't currently have a standard on whether to end with punctuation or not and the function above has punctuation

Reply:
huh, not sure how this one even got changed